recipe/meta.yaml

```diff
@@ -54,6 +54,7 @@ outputs:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('fortran') }}  # [not win]
+        - {{ stdlib("c") }}
```

Comment:
What is this for? Is this dependency necessary for all builds or only the openmpi one? In the latter case, maybe better to add the same conditional as we have behind the `{ mpi }` lines.

Reply:
Ah great, thanks for the clarification!